world_boss/app/tasks.py

```diff
@@ -195,3 +196,17 @@ def upload_balance_result(balance: List[str], channel_id: str):
     balance_str = "\n".join(balance)
     msg = f"world boss pool balance.\naddress:{signer.address}\n\n{balance_str}"
     send_slack_message(channel_id, msg)
+
+
+@celery.task()
+def stage_transactions_with_countdown(headless_url: str, nonce_list: List[int]):
+    chord(
+        stage_transaction.signature(
```

Comment:
What happens if some of staging transaction action is failed?
Based on my understood, `send_slack_message()` will be executed only when all of the txs are succeed.